gtkcord/components/emojis/emojis.go

```diff
@@ -103,7 +103,7 @@ func (s *Spawner) newPicker(r gtk.IWidget, currentGuild discord.Snowflake) *Pick
 	picker.ShowAll()
 
 	// Make all guild pages
-	e, err := s.state.Emoji.Get(currentGuild)
+	e, err := s.state.Emojis(currentGuild)
```

Comment:
FYI `s.state.Emoji` pointed to the old emoji state that should now be `s.state.EmojiState` in ningen. The old `Get` method is still [here](https://godoc.org/github.com/diamondburned/ningen/states/emoji#State.Get).

Reply:
I messed up on this, I changed it back to use the ningen state.